tests/build_env_test.py

```diff
@@ -171,6 +171,7 @@ def test_build_env(mocker, capsys):
 
     captured = capsys.readouterr()
     opence._main(["build", build_env.COMMAND, env_file])
+    validate_conda_env_files()
```

Comment:
Shall we rename this to - `validate_and_remove_conda_env_files()` so that removal is more obvious?

Reply:
Yeah, I think that makes sense